### ISSUE TITLE
[Bug]: Lightstone not considered stone?

### ISSUE THREAD

**Issue by githubuser1256** [FTBTeam/FTB-Modpack-Issues#1235]
### Modpack

FTB Plexiglass Mountain

### Modpack version

1.11 Beta

### Describe the bug

Lightstone is not voided in the builder quarry when stone is selected with tags.

### Steps to reproduce

Build builder quarry. Select stone to be voided. Click on tags. Start quarry. See lightstone be quarried.

### Expected behaviour

Lightstone should be voided

**Comment by Raidobw2** [comment]
Sure, I'll add a tag to it to make it stone like other blocks

**Comment by Raidobw2** [comment]
Fixed in-dev for the quarry, just tested it. I added both the normal lightstone found underground as well as the hardened variant (obtained through smelting in a furnace).

For consistency these blocks are also an alternative to minecraft stone blocks in crafting. In the upcoming patch one could go caving and bring back this stone resource to make for example, redstone repeaters, machine frames, etc.
